packages/styles/src/core/layout/toolbar/_main.scss

```diff
@@ -1,36 +1,39 @@
+$_toolbar-main__size: 13 * $u1;
+
 .as-toolbar-main {
   display: flex;
-  height: 52px;
-  background-color: var(--brand-02-color, $brand-02-color);
-
-  &--bg-brand-1 {
-    background-color: var(--brand-01-color, $brand-01-color);
-  }
-
-  &--bg-brand-2 {
-    background-color: var(--brand-02-color, $brand-02-color);
-  }
+  height: $_toolbar-main__size;
+  background-color: var(--toolbar-main-bg-color, $brand-02-color);
```

Comment:
This is sooo good, we need to define good CSS Variable names to let the users override it.